docs/src/pages/components/passwordfield/react.mdx

```diff
@@ -318,12 +253,12 @@ To override styling on all TextField primitives, you can set the Amplify CSS var
 
```

Comment:
In the Target Classes section, I don't see CSS variables listed. Is this expected?

Reply:
There aren't any specific design tokens for the password field, I added in some text saying this when no specific variables are found, but maybe we should just hide the tab if none exist? Not sure which is a better user experience.